indico/modules/events/surveys/controllers/display.py

```diff
@@ -84,6 +88,7 @@ def _checkParams(self, params):
                        .options(joinedload('submissions'),
                                 joinedload('sections').joinedload('children'))
                        .one())
+        self.submission = session.user.survey_submissions.filter_by(survey=self.survey, is_submitted=False).first()
```

Comment:
:warning: This will fail if the user is not logged in (which is perfectly valid if the survey doesn't require a user)